cloudinit/config/cc_lxd.py

```diff
@@ -73,6 +74,84 @@
                 domain: lxd
             """
         ),
+        dedent(
+            """\
+            # For more complex non-iteractive LXD configuration of networks,
+            # storage_pools, profiles, projects, clusters and core config,
+            # `lxd:preseed` config can be passed directly to the command:
```

Comment:
Nit: Could we clarify that this is getting passed to `stdin` of `lxd init --preseed`? In the doc here, the command `lxd init --preseed` without the pipe in the example leaves that ambiguous.